doc/asciidoc/pagerank.adoc

```diff
@@ -1,107 +1,109 @@
+It can help find doctors or providers that are behaving in an unusual manner, and then feed the score into a machine learning algorithm.
 
-There are many more use cases which you can read about in David Gleich's https://arxiv.org/pdf/1407.5107.pdf[PageRank beyond the web^]
+There are many more use cases, which you can read about in David Gleich's https://arxiv.org/pdf/1407.5107.pdf[PageRank beyond the web^]
 // end::use-case[]
 
-== Constraints / when not to use it
+== Constraints - when not to use the PageRank algorithm
 
 // tag::constraint[]
-There are some things to be aware of when using the Page Rank algorithm.
+There are some things to be aware of when using the PageRank algorithm:
 
-* Spider traps - if there are no links from within the group to outside the groups then the group of pages is a spider trap.
+* If there are no links from within a group of pages to outside of the group, then the group is considered a spider trap.
 
-* Rank sink - this occurs when a network of pages form an infinite cycle.
+* Rank sink can occur when a network of pages form an infinite cycle.
 
-* Dead-ends and Dangling links - Dead-ends occurs when pages have no out links.
-If a page contains a link to another page which has no out links, the link would be known as a dangling link.
+* Dead-ends occur when pages have no out-links.
+If a page contains a link to another page which has no out-links, the link would be known as a dangling link.
 
-If you see unexpected results from running the algorithm it's worth doing some exploratory analysis of the graph to see if any of these problems are the cause.
-You can read http://www.cs.princeton.edu/~chazelle/courses/BIB/pagerank.htm[The Google Pagerank Algorithm and How It Works^] to learn more.
+If you see unexpected results from running the algorithm, it is worth doing some exploratory analysis of the graph to see if any of these problems are the cause.
+You can read http://www.cs.princeton.edu/~chazelle/courses/BIB/pagerank.htm[The Google PageRank Algorithm and How It Works^] to learn more.
 // end::constraint[]
 
-== Algorithm explanation on simple sample graph
+== Sample
+
+This sample will explain the PageRank algorithm using a simple graph:
 
 image::pagerank.png[]
 
-.Create sample graph
+. Use the following to create a sample graph:
++
 [source,cypher]
 ----
 include::scripts/pagerank.cypher[tag=create-sample-graph]
 ----
-
-
-.Running algorithm and streaming results
++
+. Use the following to run the algorithm and stream results:
++
 [source,cypher]
 ----
 include::scripts/pagerank.cypher[tag=stream-sample-graph]
 ----
-
-
-.Running algorithm and writing back results
++
+. Use the following to run the algorithm and write back results:
++
 [source,cypher]
 ----
 include::scripts/pagerank.cypher[tag=write-sample-graph]
 ----
 
-
 // tag::stream-sample-graph-result[]
 .Results
 [opts="header",cols="1,1"]
 |===
-| name | pageRank
```

Comment:
actually, that's the returned property-name so we can't change it or have to change it in the cypher query too